extensions/2.0/Khronos/KHR_lights_environment/README.md

```diff
@@ -0,0 +1,143 @@
+
+# KHR_lights_environment
+
+## Contributors
+
+* Norbert Nopper, <mailto:nopper@ux3d.io>
+* Rickard Sahlin, <mailto:rickard.sahlin@ikea.com>
+* Gary Hsu, Microsoft, <mailto:garyhsu@microsoft.com>
+* Mike Bond, Adobe, <mailto:mbond@adobe.com>
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+This extension may use KHR_texture_basisu, KHR_texture_float_bc6h or KHR_texture_float_rgb9e5  
+
+## Overview
+This extension provides the ability to define image-based lights in a glTF scene.  
+Image-based lights consist of an environment map that represents specular radiance for the scene as well as irradiance information.  
+This can be used on it's own - ie a glTF asset with only environment map data - for a usecase where the IBL needs to be distributed.  
+It can also be used together with model data, for usecases where a model shall be displayed in a defined environment.  
+
+Many 3D tools and engines support image-based global illumination but the exact technique and data formats employed vary.  
+Using this extension, tools can export and engines can import image-based lights and the result should be highly consistent.   
+
+This extension specifies exactly one way to format and reference the environment map to be used.  
+The goals of this are two-fold.  
+First, it makes implementing support for this extension easier.  
+Secondly, it ensures that rendering of the image-based lighting is consistent across runtimes.
+
+A conforming implementation of this extension must be able to load the image-based environment data and render the PBR materials using this lighting.  
+
+This extension is based on EXT_lights_image_based, with added support for KTX2 as a carrier for [cubemap and] HDR texture formats.  
+
+The environment light is defined by a cubemap.  
+[TBD - shall a separate KHR_texture_cubemap extension be created?]
+Cubemaps can be supplied pre-filtered, by including the pre-filtered mip-levels, otherwise filtering will be performed at load time.  
+[TBD - point to relevant filtering algorithm]  
```

Comment:
Here is a reference to the theoretical prefiltering method: https://developer.nvidia.com/gpugems/gpugems3/part-iii-rendering/chapter-20-gpu-based-importance-sampling. Faster methods are used in production than this which are more approximative.